src/helpers/namespace.ts

```diff
@@ -5,6 +5,7 @@ declare global {
   namespace ${nsName} {}
 
   type DeepPartial<T> = T extends object ? { [P in keyof T]?: DeepPartial<T[P]> } : T;  
+  type Nullable<T> = T | null;
```

Comment:
```suggestion
```

I add the `DeepPartial` as a global type because it is a recursive, the nullable is simpler and a `| null` can be added in each case. Therefore we do not pollute the global scope.

Reply:
cool that's what I had originally.